### ISSUE TITLE
Update of H2_grid based on new Publication of FNB-Gas

### ISSUE THREAD

**Issue by lenzim97** [openego/powerd-data#340]
On 22.10.24 Bundesnetzagentur permit the proposal of the Wasserstoffkernetz. In this newest version there were some minor adjustments, which has to be implemented into the datamodel.

- [ ] Check how many pipelines are affacted
- [ ] best way: Build updated version on potential excel-files (not published yet, just pdf file with permitted pipelines)
- [ ] fast way: Manuell adjustments

**Comment by CarlosEpia** [comment]
@lenzim97 can we close this issue?

**Comment by lenzim97** [comment]
No, this is still open, the current version of our H2_grid is based on the publication from July 2024. If we want to update the H2-grid based on the newest plans, i would keep this issues
